### PR TITLE
Automated/release alluxio enterprise 2.2.0-1.8.0

### PR DESCRIPTION
**Pull request by madanadit**
Upgrade to latest Alluxio release

### CODE REVIEW

**Comment by madanadit** [comment]
@ryadav88 Can you take a look?

File: repo/packages/A/alluxio-enterprise/101/resource.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "assets": {
+    "uris": {
+      "alluxio-agent-darwin_386": "https://alluxio-dcos-release.s3.amazonaws.com/agent/v1.5.0/darwin_386/alluxio-dcos",
+      "alluxio-agent-darwin_amd64": "https://alluxio-dcos-release.s3.amazonaws.com/agent/v1.5.0/darwin_amd64/alluxio-dcos",
```
Comment (ryadav88) [inline repo/packages/A/alluxio-enterprise/101/resource.json:5]: @madanadit looks like its still using the old binaries?

Comment (madanadit) [reply repo/packages/A/alluxio-enterprise/101/resource.json:5]: @ryadav88 this binary is used to fetch the required alluxio version and does not need to be updated for alluxio 1.8

Comment (madanadit) [reply repo/packages/A/alluxio-enterprise/101/resource.json:5]: for air-gapped deployments, this binary is not used

**Review by ryadav88 — APPROVED** [review]

**Pull request merged**